### PR TITLE
SPARQL 1.1: Covers GROUP_CONCAT behavior with language tags

### PR DESCRIPTION
**Pull request by Tpt**
GROUP_CONCAT is defined on top of CONCAT and CONCAT has a well-defined behavior with language tag in RDF 1.1

### CODE REVIEW

File: sparql/sparql11/aggregates/agg-groupconcat-5.srx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PREFIX : <http://www.example.org/>
```
Comment (rubensworks) [inline sparql/sparql11/aggregates/agg-groupconcat-5.srx:1]: Something probably went wrong here. Should be a results file.

Comment (Tpt) [reply sparql/sparql11/aggregates/agg-groupconcat-5.srx]: Sorry for that. It seems I got mixed up in my MRs and forgot to run this one with Oxigraph. Fixed and the tests pass with Oxigraph.

**Review by rubensworks — APPROVED** [review]
Thanks for the fix!

Test 4 doesn't pass on my end, but that's my mistake :-)

**Comment by Tpt** [comment]
> Test 4 doesn't pass on my end, but that's my mistake :-)

Great news! The work to add and review these tests is not a waste!

**Pull request merged**